src/run-block-suggestion.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  BlockSuggestionInvocationBody,
+  EventTypes,
+  FunctionModule,
+  InvocationPayload,
+} from "./types.ts";
+import { UnhandledEventError } from "./run-unhandled-event.ts";
+
+export const RunBlockSuggestion = async (
+  payload: InvocationPayload<BlockSuggestionInvocationBody>,
+  functionModule: FunctionModule,
+  // deno-lint-ignore no-explicit-any
+): Promise<any> => {
+  const { body, context } = payload;
+  const env = context.variables || {};
+  const team_id = context.team_id || "";
```

Comment:
Maybe worth dropping a TODO in here to check what shape enterprise_id comes in in these events. Hopefully similarly to how it comes in for block actions: https://github.com/slackapi/deno-slack-runtime/pull/38/files#diff-e306b98ca4957a72f121f185cfd25e3084c42300d1467b06a9e31597008e5e8aR17

Reply:
I can take a peek and then add it to this PR so we can merge #38 first, and have it available for these handlers as well.